client.go

```diff
@@ -105,9 +116,20 @@ func NewFileClient(keyID string) (Client, error) {
 	return c, nil
 }
 
+//Creates a mockKeyVersion to be used for testing
+func NewMockKeyVersion(keydata []byte, status VersionStatus) KeyVersion {
+	return KeyVersion{Data: keydata, Status: status}
+}
+
 // NewMock is a knox Client to be used for testing.
 func NewMock(primary string, active []string) Client {
```

Comment:
Is this mock going to be expressive enough for you to write good tests downstream? (in your next internal change)

I am assuming you are going to actually be using the version info in testing responses and might want to just modify anything on the key.